patches/sudo-tapsetup.patch

```diff
@@ -0,0 +1,86 @@
+38,39c38,39
+<             sudo kldload if_bridge
```

Comment:
I did not test it, but is removing sudo everywhere compulsory ?

Reply:
@astralien3000 to avoid doing some Docker hacks yes, it is the simplest way to go.